apps/meteor/ee/app/livechat-enterprise/server/lib/query.helper.js

```diff
@@ -18,13 +18,18 @@ export const addQueryRestrictionsToRoomsModel = (originalQuery = {}) => {
 	return query;
 };
 
-export const addQueryRestrictionsToDepartmentsModel = async (originalQuery = {}) => {
-	const query = { ...originalQuery, type: { $ne: 'u' } };
+export const restrictQuery = async (originalQuery = {}) => {
```

Comment:
Is this still needed, considering you've already moved this logic to `applyDepartmentRestrictions.ts`